src/components/UserInfo.js

```diff
@@ -1,18 +1,21 @@
 export default class UserInfo {
-  constructor({ nameSelector, jobSelector }) {
+  constructor({ nameSelector, jobSelector, avatarSelector }) {
     this._nameElement = document.querySelector(nameSelector);
     this._jobElement = document.querySelector(jobSelector);
+    this._userAvatarElement = document.querySelector(avatarSelector);
   }
 
   getUserInfo() {
     return {
       name: this._nameElement.textContent,
-      job: this._jobElement.textContent,
+      about: this._jobElement.textContent,
```

Comment:
When I open the profile modal, there is `undefined` istead of the job text https://snipboard.io/qiQHjL.jpg


You shouldn't have changed `job` here in `getUserInfo`


```js

      job: this._jobElement.textContent,

```